app/config/routing.yml

```diff
@@ -1,3 +1,12 @@
+customer:
+    resource: "@CustomerBundle/Controller/"
+    type:     annotation
+    prefix:   /
+
+user_user:
+    resource: "@UserBundle/Controller/UserController.php"
```

Comment:
czemu to Resource jest tylko na 1 kontroler, powinno być tak jak wyżej na cały katalog